gobblin-metrics/src/main/java/gobblin/metrics/event/sla/SlaEventSubmitter.java

```diff
@@ -0,0 +1,129 @@
+package gobblin.metrics.event.sla;
+
+import java.util.Map;
+import java.util.Properties;
+
+import lombok.AccessLevel;
```

Comment:
Since there are more and more Lombok dependencies, please write a github wiki on how to setup Lombok.